api/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM golang:1.13.8 AS build-env
+MAINTAINER gxthrj@163.com
```

Comment:
,

Reply:
All source codes should belong to Apache

Reply:
done, removed